training/distributed_training/pytorch/model_parallel/mask-rcnn/smp_maskrcnn_tutorial.ipynb

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
<p>Get subnet id and security group id via boto3 </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1965/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>